opennms-base-assembly/src/main/filtered/etc/eventd-configuration.xml

```diff
@@ -1,2 +1,2 @@
-<EventdConfiguration xmlns="http://xmlns.opennms.org/xsd/config/eventd" TCPAddress="127.0.0.1" TCPPort="5817" UDPAddress="127.0.0.1" UDPPort="5817" receivers="5" getNextEventID="SELECT nextval('eventsNxtId')" socketSoTimeoutRequired="yes" socketSoTimeoutPeriod="3000">
+<EventdConfiguration xmlns="http://xmlns.opennms.org/xsd/config/eventd" TCPAddress="127.0.0.1" TCPPort="5817" UDPAddress="127.0.0.1" UDPPort="5817" receivers="5" getNextEventID="SELECT nextval('eventsNxtId')" socketSoTimeoutRequired="yes" socketSoTimeoutPeriod="3000" sink-threads="300">
```

Comment:
Can we omit the entry for `sink-threads` here? We have a default value for this attribute.